f8a_report/report_helper.py

```diff
@@ -93,26 +93,51 @@ def validate_and_process_date(self, some_date):
     def retrieve_stack_analyses_ids(self, start_date, end_date):
         """Retrieve results for stack analyses requests."""
         try:
+            # start_date from which data is to be fetched
             start_date = self.validate_and_process_date(start_date)
+            # end_date up to which data is to be fetched
             end_date = self.validate_and_process_date(end_date)
         except ValueError:
+            # checks Invalid date format
             raise ValueError("Invalid date format")
-
+        # Query to fetch Stack Analysis Ids from start_date to end_date
         query = sql.SQL('SELECT {} FROM {} WHERE {} BETWEEN \'%s\' AND \'%s\'').format(
             sql.Identifier('id'), sql.Identifier('stack_analyses_request'),
             sql.Identifier('submitTime')
         )
+        # Executing Query
         self.cursor.execute(query.as_string(self.conn) % (start_date, end_date))
-
+        # Fetching all results
         rows = self.cursor.fetchall()
-
+        # making Sorted data Lists
         id_list = []
         for row in rows:
             for col in row:
                 id_list.append(col)
 
         return id_list
 
+    def retrieve_stack_analyses_content(self, start_date, end_date):
+        """Retrieve results for stack analyses requests."""
+        try:
+            # start_date from which data is to be fetched
+            start_date = self.validate_and_process_date(start_date)
+            # end_date up to which data is to be fetched
+            end_date = self.validate_and_process_date(end_date)
+        except ValueError:
+            # checks Invalid date format
+            raise ValueError("Invalid date format")
+
+        # Query to fetch Stack Analysis manifests data from start_date to end_date
+        query = sql.SQL('SELECT {} FROM {} WHERE {} BETWEEN \'%s\' AND \'%s\'').format(
+            sql.Identifier('requestJson'), sql.Identifier('stack_analyses_request'),
+            sql.Identifier('submitTime')
+        )
+        # Executing Query
+        self.cursor.execute(query.as_string(self.conn) % (start_date, end_date))
```

Comment:
As said above, you do not need to run the same weekly query on the same table to get additional identifiers. Use the query in line no. 104 above